app/models/statement.rb

```diff
@@ -24,15 +24,19 @@ def latest_reconciliation
   end
 
   def record_actioned!
-    self.actioned_at = DateTime.now
+    self.actioned_at = Time.now
     save!
   end
 
+  def actioned?
+    !self.actioned_at.nil?
```

Comment:
This can be written in a more concise way using ActiveRecord's attribute query methods:

```ruby
actioned_at?
```

See the [`ActiveRecord::Base` docs about attribute query methods](http://api.rubyonrails.org/classes/ActiveRecord/Base.html#class-ActiveRecord::Base-label-Attribute+query+methods) for details.